app/main.py

```diff
@@ -1 +1,14 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    command_name, file_name, file_path = command.split()
+
+    if command_name != "mv" or file_path[-1] == "/":
```

Comment:
Do not use directory-specific delimiters in the code, your code should be platform-agnostic